types/jsdom/ts3.5/index.d.ts

```diff
@@ -52,7 +56,7 @@ declare module 'jsdom' {
     }
 
     class ResourceLoader {
-        fetch(url: string, options: FetchOptions): Promise<Buffer> | null;
+        fetch(url: string, options: FetchOptions): AbortablePromise<Buffer> | null;
```

Comment:
Actually, this cannot return `null`:
```suggestion
        fetch(url: string, options: FetchOptions): AbortablePromise<Buffer>;
```